scripts/ci/check_compliance.py

```diff
@@ -332,7 +332,7 @@ class DevicetreeBindingsCheck(ComplianceTest):
     """
     name = "DevicetreeBindings"
     doc = "See https://docs.zephyrproject.org/latest/build/dts/bindings.html for more details."
-    path_hint = "<zephyr-base>"
+    path_hint = "<git-top>"
```

Comment:
To make this usable for a module, the `dts` binding directory should be clearly defined instead of using the hard-coded `dts/bindings/` pattern. Preferably from the module meta data.

Reply:
It is clearly defined.

However, dts files in the module were not being analyzed without this change when this check is run from the roof of the module using T2 Star Topology.

Reply:
Ah, I see what you mean now.

Reply:
Actually, this pattern is applied to the files listed in diff output, and regardless of what the dts roots are for a module, there should still be a `dts/bindings/**/*.yaml` pattern.

So no adjustment was necessary here.